cli/contract_test.go

```diff
@@ -104,3 +104,55 @@ func TestComlileAndInvokeFunction(t *testing.T) {
 		require.Equal(t, []byte("on update|sub update"), res.Stack[0].Value())
 	})
 }
+
+func TestCompileExamples(t *testing.T) {
```

Comment:
But we already have something like that in the compiler.

Reply:
It is here mainly to cover compiler options settings. Maybe we can cut this test to just 3 examples (both tokens and contract with invalid events).